Files/Filesystem/Search/Date.cs

```diff
@@ -0,0 +1,265 @@
+﻿using Microsoft.Toolkit.Uwp;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace Files.Filesystem.Search
+{
+    public struct Date : IEquatable<Date>, IComparable<Date>, IFormattable
```

Comment:
Hmmm, what's the point of this struct?

Reply:
@d2dyno1 The interest is to manage the dates, without the hours. A native Date structure has been around for a short time in .net, but not yet in uwp. This class is scheduled to disappear when the native class becomes available. It greatly simplifies the DateRange structure, which manages ranges of dates, named or not. Of specific interest are MinValue and MaxValue which correspond to days and not hours. The exact comparison is therefore possible.